test/test_node.py

```diff
@@ -24,6 +24,58 @@ def test_inputs_are_cut_to_match_arity():
     assert node.inputs == inputs[:2]
 
 
+def _test_graph_call_and_to_x_compilations(
+    genome,
+    x,
+    y_target,
+    *,
+    skip_graph_call=False,
+    skip_to_func=False,
+    skip_to_numpy=False,
+    skip_to_torch=False,
+    skip_to_sympy=False,
```

Comment:
Kind of minor, but I would prefer to turn these switches around and rather to something like e.g. `test_to_func=True` here and then below `if test_to_func:`.
It is easier to parse if the default arg is `True` .

Reply:
yes, good suggestion!